lib/web_ui/dev/test_platform.dart

```diff
@@ -248,17 +248,8 @@ To automatically create this file call matchGoldenFile('$filename', write: true)
     );
 
     if (diff.rate > 0) {
-      // Images are different, so produce some debug info
-      final String testResultsPath = isCirrus
```

Comment:
Should we remove `isCirrus` entirely?

Reply:
later in another PR (it will probably add another 20 files :) )